packages/run-protocol/src/vpool-xyk-amm/pool.js

```diff
@@ -114,8 +114,18 @@ const helperBehavior = {
     }
     zcfSeat.exit();
     updateUpdaterState(updater, pool);
+    facets.helper.updateMetrics();
     return 'Added liquidity.';
   },
+  updateMetrics: ({ state, facets }) => {
```

Comment:
```suggestion
  /** @param {MethodContext} context */
  updateMetrics: ({ state, facets }) => {
```
this would reveal that `metricsPublication` and `metricsSubscription` need to be added to the `ImmutableState` type.

Reply:
done